transportable-udfs-spark_2.12/build.gradle

```diff
@@ -0,0 +1,52 @@
+apply plugin: 'scala'
+
+sourceSets {
+  main {
+    scala {
+      srcDirs = project(':transportable-udfs-spark_2.11').sourceSets.main.scala.srcDirs
```

Comment:
2.12 build.gradle for spark_2.11 project?

Reply:
the build.gradle file is different, but it uses the same code as spark_2.11. We dont want to copy the code unnecessarily if we can use gradle to do that.